main.rb

```diff
@@ -0,0 +1,30 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'The method has not been implemented.'
+  end
+end
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super()
```

Comment:
[OPTIONAL]
- [ ] Since you do not have any parameter in the parent `class` Nameable, you may ignore the `super` method

Reply:
But i got Linter errors if I try to remove the super method , any solution?

Reply:
> But I got Linter errors if I try to remove the super method, any solution? @christianbiring1

If that is the case you can leave it just like that